rte_mis.install

```diff
@@ -40,3 +40,47 @@ function rte_mis_install() {
   }
 
 }
+
+/**
+ * Impelments hook_schema().
+ */
+function rte_mis_schema() {
+  $schema['lottery_school_seats_status'] = [
+    'description' => 'Stores the result of the lottery',
```

Comment:
This will store the seat status for school, not the result of lottery.

Reply:
ok